src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -17,44 +17,42 @@
 namespace Akka.Remote.Transport
 {
     /// <summary>
-    /// Transport implementation used for testing.
-    /// 
-    /// The TestTransport is basically shared memory between actor systems. It can be programmed to emulate
-    /// different failure modes of a <see cref="Transport"/> implementation. TestTransport keeps a log of the activities
-    /// it was requested to do. This class is not optimized for performance and MUST not be used in production systems.
```

Comment:
I ran a ReSharper reformatting of this file, so there are lots of diffs that amount of changing tabs and spacing.